fuse-connect/src/main/java/com/letsfuse/connect/FuseConnectActivity.kt

```diff
@@ -42,9 +44,27 @@ class FuseConnectActivity : Activity() {
         var onExit: ((Exit) -> Unit)? = null
     }
 
+    private fun getDeviceHostname(): String {
+        val manufacturer = Build.MANUFACTURER
+        val model = Build.MODEL
+        return "$manufacturer $model"
+    }
+
+    private fun log(message: String) {
+        LogDna.log(
```

Comment:
would this also log it on the console?

Reply:
Nope, just pushed a change so that it does